backend/src/main/java/gov/cdc/usds/simplereport/db/repository/DeviceTypeRepository.java

```diff
@@ -17,5 +17,6 @@ public interface DeviceTypeRepository extends EternalAuditedEntityRepository<Dev
         "supportedDiseaseTestPerformed",
         "supportedDiseaseTestPerformed.supportedDisease"
       })
-  DeviceType findDeviceTypeByModelIgnoreCase(String model);
+  @Override
+  List<DeviceType> findAll();
```

Comment:
findAll() should be there by default, No need to add it here, Note that findAll will get all devices. including soft deleted ones.

also I noticed this PR and the [device sync PR](https://github.com/CDCgov/prime-simplereport/pull/5491/files#diff-43777c047ff4887bd64bddebfd137556743e2b642cd09705d0ed048c1b48489f) are removing this query